Jenkinsfile

```diff
@@ -35,7 +35,7 @@ final def mavens = env.BRANCH_NAME == 'master' ? ['3.6.x', '3.2.x'] : ['3.6.x']
 // all non-EOL versions and the first EA
 final def jdks = [14, 13, 11, 8, 7]
 
-final def options = ['-e', '-V', '-B', '-nsu', '-P', 'run-its']
+final def options = ['-e', '-V', '-B', '-nsu']
```

Comment:
Are you sure?

Reply:
Maybe wanted to avoid some annoying build failures that time ;-)
But now it is okay. Pls see the Jenkinsfile again. There is wider history of artifacts, statistics enabled in Jenkinsfile.